### DISCUSSION TITLE
Jupyter notebook

### DISCUSSION THREAD

**Discussion by witchdoctor836** [Security-Onion-Solutions/securityonion#6291] [General] [answered]
Hello Everyone!

Are there any Security Onion professionals that have successfully intergraded jupyter notebooks to work with their SOv2 SIEM?

**Comment by weslambert** [comment, marked as answer]
> 

I experimented with this on a previous version of the Elastic stack, when we weren't requiring auth/RBAC. You'll want to be careful about opening up your cluster from the firewall perspective, and you'll need to be mindful of the authentication details. Other than that, it should be pretty straightforward using the  `elasticsearch` library for Python.

**Reply by ColeVan** [reply]
So perhaps I should make a read only user for elastic and use those creds to connect to elastic on port 9200.... right?

**Reply by dougburks** [reply]
@weslambert just added a new page to our documentation for this:
https://docs.securityonion.net/en/2.3/jupyter.html

**Reply by witchdoctor836** [reply]
It helped alot. Thanks you guys! Your team is amazing!!!
